packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -92,11 +102,14 @@ export const getL2TestnetNetworkConfig = (): NetworkConfig => {
 export const ERC20TokenName = 'Test Arbitrum Token'
 export const ERC20TokenSymbol = 'TESTARB'
 export const ERC20TokenDecimals = 18
-export const invalidTokenAddress = '0x0000000000000000000000000000000000000000'
+export const invalidTokenAddress = utils.computeAddress(utils.randomBytes(32))
 
-export const zeroToLessThanOneETH = /0(\.\d+)*( ETH)/
 export const moreThanZeroBalance = /0(\.\d+)/
 
+export function getZeroToLessThanOneNativeToken(symbol: string) {
+  return new RegExp(`0(\\.\\d+)*( ${symbol})`)
```

Comment:
Since the native token is always generated from `Cypress.env('NATIVE_TOKEN_SYMBOL')`, should we keep `zeroToLessThanOneETH` as is and have `zeroToLessThanOneNativeToken` use this value internally?

Reply:
I don't think we need to maintain 2 methods, but I renamed it to `getZeroToLessThanOneToken` because it can support any token symbol